game/scripts/script.rpy

```diff
@@ -1,3 +1,4 @@
 ﻿label start:
-    $ renpy.movie_cutscene("images/intro.mp4")
+    $ renpy.movie_cutscene("images/intro.mp4") # this still doesn't work for me. I checked ren'py.org and 
+                                            # it didnt list mp4 as acceptable. Could that be the issue? do we have to conver to webm?
```

Comment:
MP4 is a container format for MPEG data streams, which should be okay. I still have to double-check if the audio and video codecs are correct now, but if push comes to shove I'll convert it to WEBM instead